packages/www/hooks/use-api/endpoints/stream.ts

```diff
@@ -61,12 +62,56 @@ export const getStreams = async (
       streamsonly: 1,
     })}`
   );
+
+  const [allStreamRes] = await context.fetch(
+    `/stream?${qs.stringify({
+      userId,
+      filters,
+      active: undefined,
+      limit: opts?.limit,
+      count: true,
+      streamsonly: 1,
+    })}`
+  );
+  const [activeStreamRes] = await context.fetch(
+    `/stream?${qs.stringify({
+      userId,
+      filters,
+      active: true,
+      limit: opts?.limit,
+      count: true,
+      streamsonly: 1,
+    })}`
+  );
+
+  const [unHealtyStreamRes] = await context.fetch(
+    `/stream?${qs.stringify({
+      userId,
+      filters,
+      active: true,
+      isHealthy: false,
+      limit: opts?.limit,
+      count: true,
+      streamsonly: 1,
+    })}`
+  );
+
   if (res.status !== 200) {
     throw new Error(streams);
   }
   const nextCursor = getCursor(res.headers.get("link"));
   const count = res.headers.get("X-Total-Count");
-  return [streams, nextCursor, count];
+  const allStreamCount = allStreamRes.headers.get("X-Total-Count");
+  const activeStreamCount = activeStreamRes.headers.get("X-Total-Count");
+  const unHealtyStreamCount = unHealtyStreamRes.headers.get("X-Total-Count");
+  return [
+    streams,
+    nextCursor,
+    count,
+    allStreamCount,
+    activeStreamCount,
```

Comment:
@adamsoffer Just wanted to confirm if this approach is fine. To make it better, we can also use promise.all for all those 3 api requedst